src/docs/sdk/unified-api/tracing.mdx

```diff
@@ -67,42 +58,89 @@ detail. Semantically, transactions represent both the top-level span of a span
+3. If `tracesSampler` is not defined, but there's a parent sampling decision, the parent sampling decision will be used.
+4. If `tracesSampler` is not defined and there's no parent sampling decision, `tracesSampleRate` will be used.
+
+<Alert title="Note" level="info">
+<markdown>
+
+Transactions should be sampled only by `tracesSampleRate` or `tracesSampler`. The `sampleRate` configuration is used for error events and should not apply to transactions.
+
+</markdown>
+</Alert>
+
+### Sampling Context
+
+If defined, the `tracesSampler` callback should be passed a `samplingContext` object, which should include, at minimum:
+
+- The `transactionContext` with which the transaction was created
+- A boolean `parentSampled` which contains the sampling decision passed down from the parent, if any
+- Data from an optional `customSamplingContext` object passed to `startTransaction` when it is called manually
+
+Depending on the platform, other default data may be included. (For example, for server frameworks, it makes sense to include the `request` object corresponding to the request the transaction is measuring.)
+
+### Propagation
+
+A transaction's sampling decision should be passed to all of its children, including across service boundaries. This can be accomplished in the `startChild` method for same-service children and using the `senry-trace` header for children in a different service.
 
 ## Header `sentry-trace`
 
-`sentry-trace = traceid-spanid-sampling`
+`sentry-trace = traceid-spanid-sampled`
 
-With sampling being optional. So at a minimum, it's expected:
+`sampled` is optional. So at a minimum, it's expected:
 
 `sentry-trace = traceid-spanid`
 
-To offer a minimal compatibility with W3C `traceparent` (without the version prefix) and `b3`  (which considers valid both 64 and 128 bits for `traceId`) headers the propagation header should have a `traceId` of 128 bits encoded in 32 hex chars and a `spanId` of 64 bits encoded in 16 hex chars.
-To avoid confusion with W3C `traceparent` due to being similar but not the exactly an implementation of it, we call it simply `sentry-trace`.
+To offer a minimal compatibility with the [W3C `traceparent` header](https://www.w3.org/TR/trace-context/#traceparent-header) (without the version prefix) and [Zipkin's `b3` headers](https://zipkin.io/pages/instrumenting#communicating-trace-information) (which consider both 64 and 128 bits for `traceId` valid), the `sentry-trace` header should have a `traceId` of 128 bits encoded in 32 hex chars and a `spanId` of 64 bits encoded in 16 hex chars.
+To avoid confusion with the W3C `traceparent` header (to which our header is similar but not identical), we call it simply `sentry-trace`.
 No version is being defined in the header.
 
-### Sampling
+### `sampled` Value`
```

Comment:
Code formatting [renders](https://develop-git-kmclb-add-tracessampler.sentry.dev/sdk/unified-api/tracing#sampled-value) quite badly on headers, I'm going to revert this.

![image](https://user-images.githubusercontent.com/88819/95197462-f08cf780-07d9-11eb-9f22-8d8e224357d1.png)

Reply:
Great, thanks!